src/routes/etudiants/+page.svelte

```diff
@@ -0,0 +1,31 @@
+<script lang="ts">
+	import type { PageData } from '../etudiants/$types';
+	import Button from '$lib/components/Button.svelte';
+
+	export let data: PageData;
+</script>
+
+<div>
+	<ul>
+		{#if data.profile}
+			{#each data.profile as user}
```

Comment:
Destructure `user` like so
```suggestion
			{#each data.profile as {user_id, first_name, last_name}
```
to avoid repeating `user.`